Parsely/wp-parsely
# Pull the Recommended Content Widget script into a JS file

## Description

Move the front end code from the PHP file to a new JavaScript file in the src directory & include it in the build step.

## Motivation and Context

Part of the larger goal to pull JS out of PHP:
Fixes #158.

## How Has This Been Tested?

* Add a known Parse.ly site URL to hosts file
* Configure your server to answer requests to that domain (for example, in [Local](https://localwp.com/), set the `Site Domain`)
* Configure the `Parse.ly Site ID` to match the site URL above
* Add & configure a widget 
* Browse to a page that loads the widget
  * Confirm the experience is equivalent on this branch and `origin/develop`

## Screenshots (if appropriate):

e.g.

![Screen Shot 2021-05-05 at 10 47 59 PM](https://user-images.githubusercontent.com/1587282/117235080-1a4fe080-adf4-11eb-9aac-1071871d5bb4.png)
(before & after should be indistinguishable)
